com.github.Bleuzen.FFaudioConverter.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.Bleuzen.FFaudioConverter</id>
+  <launchable type="desktop-id">ffaudioconverter.desktop</launchable>
+  <project_license>MIT</project_license>
```

Comment:
Need `metadata_license` also. `CC0-1.0` recommended.

Reply:
I have added it